app/components/WalletProvider.tsx

```diff
@@ -34,19 +34,20 @@ export default function WalletProvider({ children, network }: { children: ReactE
       new MetaMaskConnector({ chains }),
       // TODO implement COINBASE WALLET
       new CoinbaseWalletConnector({
-        chains,
+        chains: chains,
```

Comment:
curious to know why did this need to change? or did you just change while debugging?